templates/shared/issueicon.tmpl

```diff
@@ -1,15 +1,15 @@
 {{if .IsPull}}
 	{{if and .PullRequest .PullRequest.HasMerged}}
 		{{svg "octicon-git-merge" 16 "text purple"}}
-	{{else if and (.GetPullRequest ctx) (.GetPullRequest ctx).HasMerged}}
+	{{else if and (.PullRequest) (.PullRequest.HasMerged)}}
```

Comment:
The condition is the same as in line 2, so you can remove the whole `else if` branch.

Reply:
Done. It's weird how the previous code was added.